tasmota/tasmota_template.h

```diff
@@ -347,6 +348,7 @@ const char kSensorNames[] PROGMEM =
   D_SENSOR_MAX7219_CLK "|" D_SENSOR_MAX7219_DIN "|" D_SENSOR_MAX7219_CS "|"
   D_SENSOR_TFMINIPLUS_TX "|" D_SENSOR_TFMINIPLUS_RX "|"
   D_SENSOR_ZEROCROSS "|"
+  D_SENSOR_MCP2515_CS "|"
```

Comment:
Ditto

Reply:
Moved to below D_SENSOR_INTERRUPT at end of list